src/components/look-controls.js

```diff
@@ -144,9 +149,11 @@ module.exports.Component = registerComponent('look-controls', {
     canvasEl.removeEventListener('mouseout', this.onMouseUp);
 
     // Touch events.
-    canvasEl.removeEventListener('touchstart', this.onTouchStart);
-    canvasEl.removeEventListener('touchmove', this.onTouchMove);
-    canvasEl.removeEventListener('touchend', this.onTouchEnd);
+    if (this.data.touchEnabled) {
+      canvasEl.removeEventListener('touchstart', this.onTouchStart);
+      canvasEl.removeEventListener('touchmove', this.onTouchMove);
+      canvasEl.removeEventListener('touchend', this.onTouchEnd);
+    }
```

Comment:
It may be better to return early from these event listeners when touch is disabled, rather than conditionally binding or not. Otherwise, (1) `setAttribute('look-controls', {touchEnabled: false})` will have no effect, and (2) calls to `pause()` after that will fail to remove the old touch listeners.

So instead:

```js
onTouchStart: function () {
  if (!this.data.touchEnabled) { return; }

  // ...
}
```

Reply:
Addressed in commit 1d6930b61ac66d080bfe57eada841a032fd6e6e8